android/app/build.gradle

```diff
@@ -133,12 +133,11 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.zecwalletmobile"
+        applicationId 'com.ZingoMobile'
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 01
+        versionCode 05
```

Comment:
Why 05?

Reply:
Because I uploaded to google play 5 different versions already.